collect_app/src/main/java/org/odk/collect/android/activities/FileManagerTabs.java

```diff
@@ -14,83 +14,39 @@
- */
-public class FileManagerTabs extends TabActivity {
+public class FileManagerTabs extends FragmentActivity {
 
-    private static final String FORMS_TAB = "forms_tab";
-    private static final String DATA_TAB = "data_tab";
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setTitle(getString(R.string.manage_files));
 
-        final TabHost tabHost = getTabHost();
-        tabHost.getTabWidget().setBackgroundColor(Color.DKGRAY);
-
-        Intent remote = new Intent(this, DataManagerList.class);
-        tabHost.addTab(tabHost
-                .newTabSpec(DATA_TAB)
-                .setIndicator(getString(R.string.data))
-                .setContent(remote));
-
-        Intent local = new Intent(this, FormManagerList.class);
-        tabHost.addTab(tabHost
-                .newTabSpec(FORMS_TAB)
-                .setIndicator(getString(R.string.forms))
-                .setContent(local));
-
-        // hack to set font size
-        LinearLayout ll = (LinearLayout) tabHost.getChildAt(0);
-        TabWidget tw = (TabWidget) ll.getChildAt(0);
+        setContentView(R.layout.file_manager_layout);
 
-        int fontsize = Collect.getQuestionFontsize();
-
-        ViewGroup rllf = (ViewGroup) tw.getChildAt(0);
-        TextView savedFormTab = getTextViewChild(rllf);
-        if (savedFormTab != null) {
-            savedFormTab.setTextSize(fontsize);
-            savedFormTab.setTextColor(Color.WHITE);
-            savedFormTab.setPadding(0, 0, 0, 6);
-            savedFormTab.setSingleLine();
-        }
-
-        ViewGroup rlrf = (ViewGroup) tw.getChildAt(1);
-        TextView blankFormTab = getTextViewChild(rlrf);
-        if (blankFormTab != null) {
-            blankFormTab.setTextSize(fontsize);
-            blankFormTab.setTextColor(Color.WHITE);
-            blankFormTab.setPadding(0, 0, 0, 6);
-            blankFormTab.setSingleLine();
-        }
-    }
+        String tabNames[] = {getString(R.string.data), getString(R.string.forms)};
+        // Get the ViewPager and set it's PagerAdapter so that it can display items
+        ViewPager viewPager = (ViewPager) findViewById(R.id.pager);
+        viewPager.setAdapter(new ViewPagerAdapter(getSupportFragmentManager(), tabNames, 2));
 
-    private TextView getTextViewChild(ViewGroup viewGroup) {
-        for (int i = 0; i < viewGroup.getChildCount(); i++) {
-            View view = viewGroup.getChildAt(i);
-            if (view instanceof TextView) {
-                return (TextView) view;
-            }
-        }
-        return null;
+        // Give the SlidingTabLayout the ViewPager
+        SlidingTabLayout slidingTabLayout = (SlidingTabLayout) findViewById(R.id.tabs);
+        // Attach the view pager to the tab strip
+        slidingTabLayout.setDistributeEvenly(true);
+        slidingTabLayout.setFontColor(android.R.color.white);
+        slidingTabLayout.setBackgroundColor(Color.DKGRAY);
```

Comment:
That's a good question. I think we do not want to hardcode them because in the future when we want to introduce theme, the color will always be DKGRAY instead of following the theme.